styles.css

```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 15px;
+  background-color: rgb(164, 255, 225);
+}
+
+h1 {
+  color: blue;
+  text-align: center;
+  margin: inherit;
+}
```

Comment:
- [ ] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
![Capture](https://user-images.githubusercontent.com/93598189/197765215-b499f339-633e-4a34-a65f-8a5b9bbd5558.PNG)

Reply:
Thank you for your generous feedback I made the changes you suggested.